diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+I don't know
 
```

Comment:
/Users/grace/wdi/try

That last line is the key 
```bash
Graces-MacBook-Pro:try grace$
```

the last line of output is always the current directory. Afer the prompt you see try (username).